python/pyspark/pandas/sql_processor.py

```diff
@@ -300,3 +300,33 @@ def _convert_var(self, var) -> Any:
         if isinstance(var, (tuple, range)):
             return self._convert_var(list(var))
         raise ValueError("Unsupported variable type {}: {}".format(type(var).__name__, str(var)))
+
+
+def _test():
+    import os
+    import doctest
+    import sys
+    from pyspark.sql import SparkSession
+    import pyspark.pandas.sql_processor
```

Comment:
Renamed the file name from `sql.py` to `sql_processor.py` to avoid the name conflict between `sql` module and `pp.sql` function.